extension/src/agent/v1/prompts/base-system.ts

```diff
@@ -514,17 +583,57 @@ If you want to run a server, you must use the server_runner_tool tool, do not us
 
 # WRITE_TO_FILE (CRITICAL YOU MUST NEVER INST):
 You shouldn't never call read_file again, unless you don't have the content of the file in the conversation history, if you called write_to_file, the content you sent in <write_to_file> is the latest, you should never call read_file again unless the content is gone from the conversation history.
-You should never truncate the content of a file, always return the complete content of the file in your, even if you didn't modify it.
+You should never truncate the content of a file, when writing a new file, always return the complete content of the file in your, even if you didn't modify it (unless the file exists, then you must return only the udiffs).
```

Comment:
Not sure if this is good wording, might need to be "when writing to a new file output the full content" , "when editing a file output a udiff"